### PR TITLE
Revert utf8 chars and delete unnecessary files

### PR DESCRIPTION
**Pull request by tomdonaldson**
* Remove tthdef.tex for the reasons described in ivoa-std/ivoatex#65, then revert the UTF-8 usage from #11 which were working around that issue.
* Remove moc.lof and moc.lot as mentioned in #13 since they are build products, not needed and tend to cause conflicts.
* Remove a zip file that was inadvertently added to the repo after a `make package`.

* Fix the failure in building `role_diagram.pdf`.  It is not buildable in default environments because it requires `inkscape`, so instead an already-built copy is included in the repo.  However the Makefile attempts to rebuild it if `role_diagram.svg` appears to be newer than `role_diagram.pdf`.  This repository did not contain `role_diagram.svg`, so it was being built each time, so I've added it to the repo.
  * I also added `touch role_diagram.pdf` in the github workflows since sometimes they were attempting to rebuild `role_diagram.pdf` even though `role_diagram.svg` was present.  I haven't proven that this made a difference, but the first build that used it included the correct role diagram in the main pdf.
* Add the remaining figures to `FIGURES` in `Makefile` so that they get packaged up in the zip file.  Otherwise they will be missing from the html view on upload.

It's clear that I did not create this repo in the best way, so I think it should be reviewed more later for superfluous content.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+*.log
+*.out
+*.pdf
+*.toc
+*.lof
+*.lot
+ivoatexmeta.tex
+
+# OS
+.DS_Store
```
Comment (msdemlei) [inline .gitignore:21]: Ah... I've not thought of Macs a lot so far.  If this .DS_Store is something you'd expect MacOS to create in such directories, could you make a PR to ivoatex (it's still called svn-ignore.tex there) to add it?

Sorting into topical areas is nice, too.

**Review by msdemlei — APPROVED** [review]
Looks all good to me, comments are optional or wishlist items for other projects.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -23,6 +23,7 @@ jobs:
       run: |
         sudo apt update
         sudo apt install texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended xsltproc latexmk cm-super
+        touch role_diagram.pdf
```
[the remark this answers is not in the document]
Comment (tomdonaldson) [reply .github/workflows/build.yml]: That's a fair question.  I think this is a case of me changing too many things at once while trying to get things to work. I thought I saw it matter once, but no I doubt it. 

I removed that line now and the CI succeeded, so let's leave it out unless the problem crops up again.

File: .gitignore (hunk 1 above)
Comment (tomdonaldson) [reply .gitignore:21]: Good idea.  I created ivoa-std/ivoatex#69. 

Thanks for the review!

**Review by pferniqu — APPROVED** [review]
Approved

**Pull request merged**